soft4pes/model/grid/rl_grid.py

```diff
@@ -0,0 +1,163 @@
+""" Model of a grid with stiff voltage source and RL-load in alpha-beta frame"""
+
+import numpy as np
+from soft4pes.util.conversions import abc_2_alpha_beta
+
+
+class RLGrid:
+    """
+    Model of a grid with stiff voltage source and RL-load in alpha-beta frame. 
+
+    Attributes
+    ----------
+    Vgr : float
+        Grid rated voltage [pu].
+    wg : float
+        Grid angular frequency [pu].
+    Rg : float
+        Resistance [pu].
+    Xg : float
+        Reactance [pu].
+    x : 1 x 2 ndarray of floats
+        Current state of the grid [pu]. The grid state is represented by the 
+        grid current in alpha-beta frame.
+    """
+
+    def __init__(self, Vgr, fgr, Rg, Lg, base):
+        """
+        Initialize a RLGrid instance. Set inital state to zero.
+
+        Parameters
+        ----------
+        Vgr : float
+            Grid rated voltage [V].
+        fgr : float
+            Grid rated frequency [Hz].
+        Rg : float
+            Resistance [Ohm].
+        Lg : float
+            Inductance [H].
+        base : base value object
+            Base values.
+        """
+        self.Vgr = Vgr / base.V
+        self.wg = 2 * np.pi * fgr / base.w
+        self.Rg = Rg / base.Z
+        self.Xg = Lg / base.L
+        self.x = np.array([0, 0])
+        self.base = base
+
+    def get_next_state(self, x, u, v_dc, t, Ts):
```

Comment:
This function is used by the simulation and FSC-MPC.  Simulation and controller can call this function regardless of the formulation of the system., so it acts as a public interface. The state space model can, for example, have the disturbance term in some of the systems, while in others it will not have that.

When simulation is changed to continuous time, the role of the function can be revised.

Reply:
As I pointed out, this information is used in the controller (or the simulated system). But it does not have to do with the model